README.md

```diff
@@ -45,4 +60,9 @@ After deploying this, the following steps must be completed.
 | name | The name of the app/resource this code builds | `string` | n/a | yes |
 | ou | The name of the AWS OU that contains the AWS Accounts that are managed by this code. | `string` | n/a | yes |
 | tfinput\_ssm\_path | Standard prefix in Parameter Store for inputs to Terraform for initial infrastructure builds | `string` | n/a | yes |
-| tfoutput\_ssm\_path | Standard prefix in Parameter Store for generic Terraform outputs (typically needed by other infra code) | `string` | n/a | yes |
\ No newline at end of file
+| tfoutput\_ssm\_path | Standard prefix in Parameter Store for generic Terraform outputs (typically needed by other infra code) | `string` | n/a | yes |
+
+## Outputs
+
+No outputs.
+<!-- END_TF_DOCS -->
```

Comment:
Do we want folks to be able to add more notes to the README after the auto-injected text from `terraform-docs`? Might be worth a comment here making that clear.

Reply:
Added that up before this section.